io.github.clash_for_windows.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.clash_for_windows
```

Comment:
based on the git URL this project is pulled from, this app-id is likely breaking the rules as it is missing "github organiation" part in the id.

Reply:
I'm sorry that I mistakenly deleted the current branch when I changed the application name, I will fix each of the issues you mentioned and re-initiate a pull request. Thanks again for your suggestion!